packages/dd-trace/src/config.js

```diff
@@ -163,6 +164,36 @@ class Config {
       process.env.DD_TRACE_EXPERIMENTAL_TRACEPARENT_ENABLED,
       false
     )
+    const DD_TRACE_PROPAGATION_STYLE_INJECT = coalesce(
+      options.tracePropagationStyle && options.tracePropagationStyle.inject,
+      process.env.DD_TRACE_PROPAGATION_STYLE_INJECT,
+      // TODO: change this to traceparent,datadog
+      // This can't be done as a semver-minor until tracestate support also exists
+      'datadog'
+    ).split(',').reduce((m, v) => {
+      m[v.toLowerCase()] = true
+      return m
+    }, {
+      b3: isTrue(DD_TRACE_B3_ENABLED),
+      'b3 single header': isTrue(DD_TRACE_B3_ENABLED),
+      tracecontext: isTrue(DD_TRACE_TRACEPARENT_ENABLED),
```

Comment:
Is this `tracecontext` or `traceparent`? While I prefer `tracecontext` since it's more accurate, the option should probably match the spec.

Reply:
Spec calls it tracecontext. I originally had it as traceparent due to the experimental feature calling it that. Noticed the difference later and updated it.